### PR TITLE
made err logging more consistent across zping-client/server and caugh…

### PR DESCRIPTION
**Pull request by r-caamano**
…t unhandled err in server.

### CODE REVIEW

File: example/zping-client/zping-client.go [hunk 1]
```diff
@@ -135,8 +136,8 @@ func main() {
 		//fmt.Println(file)
 		configFile, err := config.NewFromFile(file)
 		if err != nil {
```
Comment (plorenz) [inline example/zping-client/zping-client.go:138]: We usually use a pattern like: `logrus.WithError(err).Error("Problem loading config file")`

File: example/zping-server/zping-server.go [hunk 2]
```diff
@@ -72,26 +73,29 @@ func main() {
 		file := *configPtr
 		configFile, err := config.NewFromFile(file)
 		if err != nil {
-			logger.Errorf("Problem loading configfile: %+v\n", err)
-			panic(err)
+			logrus.Errorf("Problem loading configfile: %+v\n", err)
```
Comment (plorenz) [inline example/zping-server/zping-server.go:76]: `os.Exit(1)`?

Comment (r-caamano) [reply example/zping-server/zping-server.go:76]: If you don't use panic the logrus just print and continues

File: example/zping-client/zping-client.go (hunk 1 above)
Comment (r-caamano) [reply example/zping-client/zping-client.go:138]: This is pattern used in chat-server
if err != nil {
		logrus.Errorf("Error binding service %+v", err)
		panic(err)
	}

Comment (r-caamano) [reply example/zping-client/zping-client.go:138]: Will change to pattern as suggested

**Pull request closed without merging**